build.gradle

```diff
@@ -1,9 +1,9 @@
 plugins {
     id 'java'
+    id 'com.adarshr.test-logger' version '3.2.0'
 }
 
 group 'com.surrealdb'
-version '0.1.0'
```

Comment:
We definitely want to keep the version 😄 I'm not actually sure what it defaults to 🤔

Reply:
In `release.yml` we build and ceate a release in Gihub Release page with the same version of the tag that triggered this workflow. This means a new release will be publish automatically on new tags. Having the version hardcoded into the source code makes updaing it a manual work. However, I see your point, so please read my other comment on `release.yml`.